config/typescript.js

```diff
@@ -1,19 +1,24 @@
 /**
- * Adds `.jsx`, `.ts` and `.tsx` as an extension, and enables JSX/TSX parsing.
+ * This config:
+ * 1) adds `.jsx`, `.ts`, `.cts`, `.mts`, and `.tsx` as an extension
+ * 2) enables JSX/TSX parsing
  */
 
-// Omit `.d.ts` because 1) TypeScript compilation already confirms that
-// types are resolved, and 2) it would mask an unresolved
-// `.ts`/`.tsx`/`.js`/`.jsx` implementation.
-const allExtensions = ['.ts', '.tsx', '.js', '.jsx'];
+/**
+ * We omit `.d.ts` because:
+ * 1) TypeScript compilation already confirms that types are resolved
+ * 2) it would mask an unresolved `.ts`/`.tsx`/`.js`/`.jsx` implementation
+ */
```

Comment:
please revert the comment changes; make sure you're not running prettier on a project that doesn't use it :-)

Reply:
I didn't use Prettier. I intentionally turned the comment into a JSDoc as a "fix", following the general heuristic of having comments that are next to a variable always be JSDoc so that they are properly "attached" to the variable. In VSCode, when you hover over the variable elsewhere in the code, it will show the comment, but when you use a line comment like this, you don't get this functionality, so using JSDoc is the more "correct" choice. Furthermore, it's a bit confusing to use both styles of comments in the file; why not choose one and stick with it?